packages/create-vite/src/index.ts

```diff
@@ -565,6 +575,34 @@ function editFile(file: string, callback: (content: string) => string) {
   fs.writeFileSync(file, callback(content), 'utf-8')
 }
 
+function showHelper() {
+  const formattedHelpText = getUsageInfo()
+  console.log(formattedHelpText)
+}
+
+function getUsageInfo() {
+  const formattedFrameworkNames = getFormattedVariantNames()
+  return `Usage: create-vite [OPTION]... [DIRECTORY]
+Create a new JavaScript project based on Vite.
+
+With no arguments, start an interactive command line dialog.
+
+Options:
+  -t, --template NAME        use this template
+
+Available templates:
+${formattedFrameworkNames.join('\n')}`
+}
+
+function getFormattedVariantNames() {
+  return FRAMEWORKS.filter(
+    (framework) => framework.name !== 'others' && framework.variants.length > 0,
+  ).map((framework) => {
+    const variantNames = framework.variants.map((variant) => variant.name)
+    return `${variantNames.join(' ')}`
+  })
+}
```

Comment:
I think we should also put the variants each on their new line as well, otherwise it may be hard to read if some are on a single line.

Also, I think we should filter out the templates that have `customCommand` as they're redirects to other CLIs.

Reply:
> I think we should also put the variants each on their new line as well, otherwise it may be hard to read if some are on a single line.
> 
> Also, I think we should filter out the templates that have `customCommand` as they're redirects to other CLIs.

```

Usage: create-vite [OPTION]... [DIRECTORY]
Create a new Vite project in JavaScript or TypeScript.

With no arguments, start an interactive command line dialog.

Options:
  -t, --template NAME        use this template

Available templates:
vanilla-ts
vanilla
vue-ts
vue
react-ts
react-swc-ts
react
react-swc
preact-ts
preact
lit-ts
lit
svelte-ts
svelte
solid-ts
solid
qwik-ts
qwik

```